packages/components/src/components/pagination/component.tsx

```diff
@@ -300,14 +300,18 @@ export class KolPagination implements API {
 					exportparts="icon"
 					key={`${this.nonce}-${page}`}
 					_customClass={this.state._customClass}
-					_label={`${page}`}
+					_label=""
 					_on={{
 						onClick: (event: Event) => {
 							this.onClick(event, page);
 						},
 					}}
 					_variant={this.state._variant}
-				></kol-button>
+				>
+					<span slot="expert">
+						<span class="visually-hidden">{translate('kol-page')}&nbsp;</span> {page}
```

Comment:
Das Leerzeichen `</span> {page}` muss noch raus.

Reply:
Wenn ich diese Leerzeichen entferne, verschiebt Prettier den '{page}' in die nächste Zeile. Sollen wir das Leerzeichen so lassen oder möchtest du, dass ich versuche, es anderes zu lösen, auch wenn '{page}' auf die nächste Zeile rutscht?

Reply:
Bitte einfach `&nbsp;` weglassen. Sonst ist es okay.